src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/comparator/CarManufacturerComparator_LowerCase.java

```diff
@@ -0,0 +1,50 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.comparator;
+
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.Comparator;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.ComparatorLogger;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.Matchable;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+//Where to find similarity measures
+//import de.uni_mannheim.informatik.dws.winter.similarity.;
+import de.uni_mannheim.informatik.dws.winter.similarity.string.GeneralisedStringJaccard;
+
+public class CarManufacturerComparator_LowerCase implements Comparator<Car, Attribute> {
+    private static final long serialVersionUID = 1L;
+
+    private GeneralisedStringJaccard sim;
+    private ComparatorLogger comparisonLog;
+
+    @Override
+    public double compare(Car record1,
+                          Car record2,
+                          Correspondence<Attribute, Matchable> schemaCorrespondence) {
+
+        if (!record1.getManufacturer().toLowerCase().equals("sonstige_autos")
```

Comment:
Will throw a NullPointerException if Manufacturer it empty. Have a look at the other comparators on how to do a null check.
And could you rename the file to follow the common scheme, e.g., CarManufacturerComparatorJaccard?